Makefile

```diff
@@ -32,10 +32,13 @@ build386:
 buildlinuxarm:
 	GOOS=linux GOARCH=arm ./tool/go install tailscale.com/cmd/tailscale tailscale.com/cmd/tailscaled
 
+buildwasm:
+	GOOS=js GOARCH=wasm ./tool/go install tailscale.com/cmd/tailscale tailscale.com/cmd/tailscaled
```

Comment:
Match the GitHub action and build `./tool/go build ./cmd/tsconnect/wasm ./cmd/tailscale/cli`